src/SAML2/XML/samlp/NameIDPolicy.php

```diff
@@ -15,7 +15,7 @@
  * @author Tim van Dijen, <tvdijen@gmail.com>
  * @package SimpleSAMLphp
  */
-final class NameIDPolicy extends \SAML2\XML\AbstractConvertable
+final class NameIDPolicy extends \SAML2\XML\AbstractXMLElement
```

Comment:
These ones are incorrect as well, they should extend from `SAML2\XML\samlp\AbstractSamlpElement`.